Omit navbar of printed pdf for UG and DG

ernestlim8 opened the pull request:
Fixes #35 

Let's add a media query for printing, to hide the navigation bar by setting the display to none.

damithc commented:
This PR/issue should be against a repo in the org [nus-cs2103-AY1617S4](https://github.com/nus-cs2103-AY1617S4) not against the repo in se-edu org :-)
Closing...

ernestlim8 committed:
Omit navbar of printed pdf for UG and DG

damithc commented:
@ernestlim8 thanks for the PR. Can add a detailed commit message as per se-edu conventions?

ernestlim8 committed:
Print to PDF shows navigation bar of the document

Navigation bar is not necessary and clutters the pdf pages

Adding a css print query will allow for the pages to be
printed out without showing the navigation bar

Let's specify the behaviour of the media print query
to change the display type of the site-header component,
which contains the navigation bar, to none

We found this fix to be the most suitable as
it only modifies the body content when
printing the document
